workspace_host/interface.js

```diff
@@ -925,6 +925,7 @@ function _pullImage(workspace, callback) {
             let current, total = 0, fraction = 0;
             let currentBase, fractionBase;
             let outputCount = 0;
+            let percentCache = 0, dateCache = Date.now();
```

Comment:
Maybe initialize these at lower values so we will always send the very first percent that we calculate?
```suggestion
            let percentCache = -1, dateCache = Date.now() - 1e6;
```

Reply:
done